activestorage/lib/active_storage/analyzer/image_analyzer/image_magick.rb

```diff
@@ -32,8 +32,10 @@ def read_image
         {}
       end
 
+      ROTATIONS = Set.new(%w[RightTop LeftBottom TopRight BottomLeft]).freeze
```

Comment:
In the case of small arrays like this one, you may find that `Array#include?` is actually faster than `Set#include?`.  (N must be sufficiently large for O(1) to beat O(N).)